### PR TITLE
DSNS-164: Updated usgs downloader with an explicit directory path for logging.

### PR DESCRIPTION
**Pull request by gordonyeong**

### CODE REVIEW

**Review by dunkgray — COMMENTED** [review]
process_log_path and logs_dir are the same. Potentially just have one.

**Comment by gordonyeong** [comment]
i did consider that but decided to do it this way because i wanted the airflow process to be more configurable (ie. we currently have both the logging directories for the actual downloader process and the airflow to be the same path now but this may change).
I am happy to just stick to one. 
Can you please confirm, Duncan? :) thank you :)

**Comment by dunkgray** [comment]
Let's have one param value for the logging.  In the future if it is needed it can be split up.

File: dags/nci_ard/nci_ls_l1_downloader.py [hunk 1]
```diff
@@ -59,9 +59,13 @@
     current_userid = "gy5636"
     params["logs_dir"] = f"/g/data/u46/users/{current_userid}/output/lpgs/"
```
Comment (dunkgray) [inline dags/nci_ard/nci_ls_l1_downloader.py:60]:
```suggestion
```

**Review by dunkgray — APPROVED** [review]
One line suggestion

File: dags/nci_ard/nci_ls_l1_downloader.py (hunk 1 above)
Comment (gordonyeong) [reply dags/nci_ard/nci_ls_l1_downloader.py:60]: yep. Sorted :)

**Comment by gordonyeong** [comment]
made change for a single log directory param.
 I have also tested

**Review by dunkgray — APPROVED** [review]

**Review by dunkgray — APPROVED** [review]

**Pull request merged**